### PR TITLE
feat(Feedback): Update animated success icon for O2

### PR DESCRIPTION
**Pull request by RubenMorPov**
[WEB-397](https://jira.tid.es/browse/WEB-397) - Changed the success icon and keeping the old animation for O2.

### CODE REVIEW

**Review by atabel — COMMENTED** [review]
I think the tick animation should happen after the circle one. See the Movistar animation

File: src/icons/icon-success.tsx [hunk 1]
```diff
@@ -28,26 +28,35 @@ const IconSuccessO2: React.FC<Props> = ({size = 64, color, skipAnimation = false
         <svg role="presentation" width={size} height={size} viewBox="0 0 64 64">
             <g
                 className={classnames({[classes.iconContainer]: !color})}
-                strokeWidth="2"
                 strokeLinecap="round"
                 strokeLinejoin="round"
             >
```
Comment (yceballost) [inline src/icons/icon-success.tsx:33]: Check animation have a little buggy translation

https://user-images.githubusercontent.com/6722153/150359271-1702ed89-c40f-4283-aaea-96b3aafc365b.mov

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/icons/icon-success.tsx (hunk 1 above)
Comment (RubenMorPov) [reply src/icons/icon-success.tsx:33]: Do you mean that little 'hop' that the check makes?

Comment (yceballost) [reply src/icons/icon-success.tsx:33]: yep!

Comment (RubenMorPov) [reply src/icons/icon-success.tsx:33]: Check if now it suits up!

Comment (yceballost) [reply src/icons/icon-success.tsx:33]: nice! currazo!

**Review by yceballost — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Pull request merged**